cristin-project/src/main/java/no/unit/nva/cristin/projects/CristinProjectBuilder.java

```diff
@@ -15,14 +19,18 @@
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.stream.Collectors;
+import no.unit.nva.utils.UriUtils;
 
 import static java.util.Objects.nonNull;
 import static no.unit.nva.language.LanguageMapper.getLanguageByUri;
+import static no.unit.nva.model.Organization.ORGANIZATION_IDENTIFIER_PATTERN;
 import static no.unit.nva.utils.ContributorRoleMapping.getCristinRole;
 import static no.unit.nva.utils.UriUtils.extractLastPathElement;
 
 public class CristinProjectBuilder {
```

Comment:
I think the conversion of the NvaContributor to a CristinPerson should be part of the NvaContributor or the CristinPerson